app/components/user-page/course-progress-list-item/index.js

```
@@ -34,6 +36,29 @@
     }
   }
 
+  get languagesToDisplay() {
+    // First, get all completed course participations sorted by completion time (most recent first)
+    const completedParticipations = this.completedCourseParticipations.sort(
+      (a, b) => new Date(b.completedAt).getTime() - new Date(a.completedAt).getTime()
```

ESLint

[error] Insert `,`

(prettier/prettier)